public/stylesheets/site/2.0/02-elements.css

```diff
@@ -67,7 +67,6 @@ h2 {
   line-height: 1;
   margin: 0.429em 0;
   display: inline;
-    text-shadow: -1px -1px 2px #fff;
```

Comment:
Yeah, it's actually a shadow that's invisible against the white background (we neglected to take it out after we got rid of the blue/lilac/grey background), but that leads to ugliness or readability issues once you start changing the background and font colors.